aten/src/ATen/native/quantized/cudnn/BinaryOps.cpp

```diff
@@ -109,8 +109,10 @@ Tensor add(Tensor qa, Tensor qb, double output_scale, int64_t output_zero_point)
   at::Tensor add_output = at::empty(qa.sizes(), at::device(at::kCUDA).dtype(at::kFloat), memory_format);
   at::Tensor quantized_output = at::_empty_affine_quantized(qa.sizes(), at::device(at::kCUDA).dtype(at::ScalarType::QInt8),
                                                             output_scale, output_zero_point, memory_format);
-  // TODO: When cudnn enables support for broadcasting, we can remove this tensor
-  at::Tensor requantize_multiplier_tensor = at::empty(quantized_output.sizes(), at::device(at::kCUDA).dtype(at::kFloat), memory_format);
+  // We will employ broadcasting scalar multiplication in cudnn in the requant_op below. For this to work, cudNN requires
```

Comment:
do we want to have a utility function for this?

Reply:
for creating a scalar tensor for an arbitrary number of dimensions? I don't think so

Reply:
@jerryzh168 for conv, `requantize_multiplier_tensor` size is known at compile time, so I use `std::array`, but for linear and add, size is not known at compile time, so I have to use `at::SmallVector`. if I make a utility function that all 3 ops can use, I would make conv's `requantize_multiplier_tensor` use ``at::SmallVector` instead of `std::array` -- probably a small hit on performance associated with this

Reply:
a utility function to create this requantize_multipler_tensor

Reply:
@jerryzh168 done. can you reapprove so I can land the stack?